Crash/empty tabs in recent chunky snapshots

```
Failed to build render controls tabs.
java.lang.NoClassDefFoundError: se/llbit/chunky/ui/RenderControlsFxController
	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3402)
	at java.base/java.lang.Class.getDeclaredMethods(Class.java:2504)
	at javafx.fxml/com.sun.javafx.fxml.BeanAdapter$1.run(BeanAdapter.java:128)
	at javafx.fxml/com.sun.javafx.fxml.BeanAdapter$1.run(BeanAdapter.java:125)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at javafx.fxml/com.sun.javafx.fxml.BeanAdapter.getClassMethodCache(BeanAdapter.java:124)
	at javafx.fxml/com.sun.javafx.fxml.BeanAdapter.<init>(BeanAdapter.java:104)
	at javafx.fxml/javafx.fxml.FXMLLoader$Element.getValueAdapter(FXMLLoader.java:207)
	at javafx.fxml/javafx.fxml.FXMLLoader$Element.getProperties(FXMLLoader.java:215)
	at javafx.fxml/javafx.fxml.FXMLLoader$PropertyElement.<init>(FXMLLoader.java:1374)
	at javafx.fxml/javafx.fxml.FXMLLoader.createElement(FXMLLoader.java:2832)
	at javafx.fxml/javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2805)
	at javafx.fxml/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2634)
	at javafx.fxml/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2548)
	at javafx.fxml/javafx.fxml.FXMLLoader.load(FXMLLoader.java:2516)
	at dev.ferrand.chunky.bloom.ui.BloomTab.<init>(BloomTab.java:43)
	at dev.ferrand.chunky.bloom.BloomPlugin.lambda$attach$0(BloomPlugin.java:24)
	at se.llbit.chunky.ui.controller.RenderControlsFxController.buildTabs(RenderControlsFxController.java:119)
	at se.llbit.chunky.ui.controller.RenderControlsFxController.<init>(RenderControlsFxController.java:93)
	at se.llbit.chunky.ui.controller.ChunkyFxController.initialize(ChunkyFxController.java:727)
	at javafx.fxml/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2655)
	at javafx.fxml/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2548)
	at javafx.fxml/javafx.fxml.FXMLLoader.load(FXMLLoader.java:2516)
	at se.llbit.chunky.ui.ChunkyFx.start(ChunkyFx.java:56)
	at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:847)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:484)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ClassNotFoundException: se.llbit.chunky.ui.RenderControlsFxController
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 34 more
```

![unknown](https://user-images.githubusercontent.com/5544859/164612716-e7366722-768c-4c76-8367-fa0adf733cee.png)

--- a/build.gradle
+++ b/build.gradle
@@ -11,7 +11,7 @@ javafx {
 apply plugin: 'java'
 
 group 'dev.ferrand'
-version '0.2.1'
+version '0.3.0'
 
 jar {
     manifest {
@@ -27,8 +27,11 @@ compileJava {
 repositories {
     mavenLocal()
     mavenCentral()
+    maven {
+        url 'https://repo.lemaik.de/'
+    }
 }
 
 dependencies {
-    compile 'se.llbit:chunky-core:2.4.0-snapshot5'
+    compile 'se.llbit:chunky-core:2.5.0-SNAPSHOT'
 }

--- a/src/main/java/dev/ferrand/chunky/bloom/ui/BloomTab.java
+++ b/src/main/java/dev/ferrand/chunky/bloom/ui/BloomTab.java
@@ -11,7 +11,7 @@ import se.llbit.chunky.PersistentSettings;
 import se.llbit.chunky.renderer.scene.Scene;
 import se.llbit.chunky.ui.DoubleAdjuster;
 import se.llbit.chunky.ui.IntegerAdjuster;
-import se.llbit.chunky.ui.RenderControlsFxController;
+import se.llbit.chunky.ui.controller.RenderControlsFxController;
 import se.llbit.chunky.ui.render.RenderControlsTab;
 import se.llbit.json.JsonNumber;
 import se.llbit.json.JsonObject;

--- a/src/main/resources/plugin.json
+++ b/src/main/resources/plugin.json
@@ -2,7 +2,7 @@
   "name": "BloomPlugin",
   "author": "a_Tom",
   "main": "dev.ferrand.chunky.bloom.BloomPlugin",
-  "version": "0.2.1",
-  "targetVersion": "2.4.0-151-g5451674f",
+  "version": "0.3.0",
+  "targetVersion": "2.5.0-62-g5719d35f",
   "description": "Add a bloom post processing filter"
 }

